CONTRIBUTING.md

```diff
@@ -20,3 +20,27 @@ Git branches and their purpose and status at the time of this writing are listed
 ## Prerequisites
 
 Follow the instructions in [README.rst](https://github.com/Pylons/pyramid_openapi3/) to install the tools needed to run the project.
+
+## Testing oldest supported versions
+
+In CI, we want to test the oldest supported versions of `openapi-core` and `pyramid` on the oldest supported Python version. We do it like so:
+
+* Have the `pyproject_py37.toml` and `poetry_py37.lock` files in this repo.
+* In the Python 3.7 CI run, `mv` them to `pyproject.toml` and `poetry.lock`.
+
+### How to prepare these two files
+
+1. In `pyproject.toml` modify the `openapi-core` and `pyramid` dependencies to be pinned to the
+    oldest supported version.
```

Comment:
Note that you have slightly inconsistent indentation. The first line starts at character 4, the next at character 5. I would indent the first lines to character 5 throughout this list. I do that as a personal preference, as it makes it easier to indent for code blocks and nested lists in most editors.

Reply:
Good point, thanks!